apps/mark/frontend/src/app_root.tsx

```diff
@@ -364,9 +364,12 @@ export function AppRoot({
 
   const hidePostVotingInstructions = useCallback(() => {
     clearTimeout(PostVotingInstructionsTimeout.current);
-    if (isCardlessVoterAuth(auth)) auth.logOut();
```

Comment:
Got rid of the old `isCardlessVoterAuth` check because `endCardlessVoterSession` is always safe to run, kind of like `logOut`. And removing the check made it a bit easier to maintain 100% code coverage